src/Filters/Filters.js

```diff
@@ -0,0 +1,171 @@
+        <input
+          type='text'
+          onChange={(e) => {
+            return setSearch(e.target.value);
+          }}
+          value={searchString ? searchString : search}
+          className='search-input-txt input-txt'
+          required
+        />
+        <span className='flt-label flt-label-search'>Search Product</span>
+        <button
+          onClick={() => dispatch({ type: "SEARCH", payload: search })}
+          className='btn-search'
+        >
+          <i className='fas fa-search'></i>
+        </button>
+      </div>
+      <h3>Filters</h3>
+      <div className='filter'>
+        Price:
+        <div>
+          <label>
+            <input
+              type='radio'
+              name='sort'
+              onChange={() => {
+                return dispatch({ type: "SORT", payload: "PRICE_LOW_TO_HIGH" });
+              }}
+              checked={sortBy && sortBy === "PRICE_LOW_TO_HIGH"}
+            />
+            Sort from Low to High
+          </label>
+        </div>
+        <div>
+          <label>
+            <input
+              type='radio'
+              name='sort'
+              onChange={() =>
+                dispatch({ type: "SORT", payload: "PRICE_HIGH_TO_LOW" })
+              }
+              checked={sortBy && sortBy === "PRICE_HIGH_TO_LOW"}
+            />
+            Sort from High to Low
+          </label>
+        </div>
+      </div>
+
+      <div className='filter'>
+        <div>
+          <label>
+            <input
+              type='checkbox'
+              checked={showInventoryAll}
+              onChange={() => dispatch({ type: "TOGGLE_INVENTORY" })}
+            />
+            Exclude Out of Stock
+          </label>
+        </div>
+        <div>
+          <label>
+            <input
+              type='checkbox'
+              checked={showFastDeliveryOnly}
+              onChange={() => dispatch({ type: "TOGGLE_DELIVERY" })}
+            />
+            Fast Delivery Only
+          </label>
+        </div>
+      </div>
+      <div className='filter'>
+        Category:
+        {categories.map((item) => {
+          return (
+            <div key={item._id}>
+              <label>
+                <input
+                  type='radio'
+                  name='sort-category'
+                  checked={item.name === category}
+                  onChange={() =>
+                    dispatch({
+                      type: "CATEGORY",
+                      payload: item.name,
+                    })
+                  }
+                />
+                {item.name}
+              </label>
+            </div>
+          );
+        })}
+      </div>
+      <div className='filter'>
+        <select
+          className='select-level'
+          value={level}
+          onChange={(e) =>
+            dispatch({ type: "SELECT_LEVEL", payload: e.target.value })
+          }
+        >
+          <option value='Beginner'>Beginner</option>
```

Comment:
I have a suggestion, create an array of level and map it here, so that it becomes easy to change the level later if needed